sem_covid/entrypoints/notebooks/EDA/eda_wrangling/cramer_analysis.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import pandas as pd
+
+from scipy.stats import chi2_contingency
+from sklearn import preprocessing
+
+
+def calc_cramer_v(var_1: pd.Series, var_2: pd.Series) -> float:
+    """
+    Function for computing Cramer's V coefficient
+    """
+    crosstab = np.array(pd.crosstab(var_1, var_2, rownames=None, colnames=None))
+    stat = chi2_contingency(crosstab)[0]
+    obs = np.sum(crosstab)
+    min_ = min(crosstab.shape) - 1
+    return stat / (obs * min_)
+
+
+def get_cramer_corr_matrix(data: pd.DataFrame) -> pd.DataFrame:
```

Comment:
do we have restrictions on the type of columns? 
what happens if we accidentally drop numeric variables :) 
I would just amend the documentation specifying this.